lexers/m/mql.go

```diff
@@ -0,0 +1,27 @@
+package m
+
+import (
+	"github.com/alecthomas/chroma"
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/c"
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// MQL lexer.
+var MQL = internal.Register(MustNewLexer(
+	&Config{
+		Name:      "MQL",
+		Aliases:   []string{"mql", "mq4", "mq5", "mql4", "mql5"},
+		Filenames: []string{"*.mq4", "*.mq5", "*.mqh"},
+		MimeTypes: []string{"text/x-mql"},
+	},
+	Rules{
+		"root": {},
+	},
+).SetAnalyser(func(text string) float32 {
```

Comment:
Let's add a test here?

Reply:
MQL text analysis inherits from C++. However the way it includes libraries seems to always have a file with extension which is not caught by C++.

Reply:
Not sure, if I understand you correctly. But this still could be tested, right? I mean the way how we test normally, by call to `AnalyseText` function, the file extension is not taken into account at all.

Reply:
Do you have an example for mql code?

Reply:
I meant C++ text aanalysis has this regex `#include <[a-z_]+>` which differs from the way that `MQL` include syntax.
https://github.com/pygments/pygments/blob/fc55dc2e95bcea03fbc0d8e1d130c9e53f3f1dad/tests%2Fexamplefiles%2Fexample.mq4#L11
https://github.com/pygments/pygments/blob/fc55dc2e95bcea03fbc0d8e1d130c9e53f3f1dad/tests%2Fexamplefiles%2Fexample.mqh#L6

So that's why I didn't add a test for it. Maybe the right thing to do is to remove the call for C++ function.

Reply:
Alright I see. Let's remove it for now :+1: And you could create a PR in pygments to adjust this. I guess mql lexer needs it's own text analysis function.